### ISSUE TITLE
minimap2 module: retain ML:B tag

### ISSUE THREAD

**Issue by cdebuck** [jbloomlab/alignparse#92]
Hello,
I'm trying to use the minimap2 module of alignparse to map reads to a genome while retaining several tags in the fastq headers.
I'm using data that contains information on modified bases in MM and ML tags.
The ML tag is of type B, and this gives an error in line 308 of minimap2.py: `ValueError: bad tag type B for tag ML`
Could the script be updated to also support tags with type B?

**Comment by jbloom** [comment]
I'm not completely certain I recommend `alignparse` as a general purpose way to run `minimap2`. But I can see where it might be useful. If you want me to do this, can you create and post a short example including the commands you are. using and a small data file with the tags you want to keep.

**Comment by cdebuck** [comment]
Hi,
I've actually found out that the  `-y` option of minimap2 is suitable for my needs, because it retains the tags that I need, so I ended up not using alignparse for my analyses. Before, I realized this I had been playing around a little with the code to try and implement retaining tags of the B-type, I will try to create pull-request to show what I had tried (I am not very experienced so I apologize if I make a mistake).
I have attached a small subset of the reads that contain the MM and ML tags that I'm interested in.

The commands I used were:

```
test_ref_file = '/home/cdebuck/ref.fasta'
test_fastq = '/home/cdebuck/sample_fastq.txt'
out_sam_file = '/home/cdebuck/test/alignparse_test_minimap.sam'
options = ['-ax', 'map-ont', '-t 16']
retain_tags = ['MM', 'ML']
mapper = minimap2.Mapper(options, check_cs=False, retain_tags=retain_tags)
mapper.map_to_sam(test_ref_file, test_fastq, out_sam_file)
```
[sample_fastq.txt](https://github.com/jbloomlab/alignparse/files/9261068/sample_fastq.txt)
